packages/react-canvas-core/src/states/DragCanvasState.ts

```diff
@@ -46,6 +46,15 @@ export class DragCanvasState extends AbstractDisplacementState {
 	}
 
 	fireMouseMoved(event: AbstractDisplacementStateEvent) {
+		const isMouseDown = event.event.buttons !== 0;
```

Comment:
i think this should be handled in the abstract displacement state (potentially).